tests/main/views/test_direct_award.py

```diff
@@ -210,6 +211,18 @@ def _cannot_start_from_task(self, tasklist, cannot_start_from):
         return all([self._task_cannot_start_yet(tasklist, task + 1) is ((task + 1) >= cannot_start_from)
                     for task in range(len(tasklist))])
 
+    def _set_project_locked_and_framework_states(self, locked_at, framework_status, following_framework_status):
+        project = self._get_direct_award_project_fixture()
+        project['project']['lockedAt'] = locked_at
+        self.data_api_client.get_direct_award_project.return_value = project
+
+        self.data_api_client.find_frameworks.return_value = {
+            'frameworks': [framework(slug='g-cloud-9', status=framework_status)['frameworks']]
+        }
+        self.data_api_client.get_framework.side_effect = [
```

Comment:
Why `side_effect` here rather than `return_value`?

Reply:
Yep. This was a hangover from a previous (wrong) version where I had two different returns - hence the list. Will change to `return_value`